test/TestUtilities/Test.Utility/MockResponses/FeedUtilities.cs

```diff
@@ -175,6 +185,19 @@ private static JObject GetPackageRegistrationItem(string serverUri, string id, s
             catalogEntry.Add(new JProperty("tags", new JArray()));
 
             return item;
+
+            static JProperty GetDeprecatedEntry()
+            {
+                return new JProperty("deprecation",
+                                    new JObject
+                                    {
+                        new JProperty("reasons", new JArray
+                        {
+                            "CriticalBugs"
+                        }),
+                        new JProperty("message", "This package is unstable and broken!")
+                                    });
```

Comment:
It's a bit hard to read because of the wild formatting.

Reply:
Ready again. 
The old one was definitely bad.